popSimManu.tex

```diff
@@ -791,23 +792,26 @@ \section*{Discussion}
 and context-dependent mutation.
 
 % note: this is addressing reviewer comments about model uncertainty
-In particular, although the included demographic histories are some of the most widely used models for our current set of species,
+We wish to emphasize that although the included demographic histories are some of the
+most widely used models for our current set of species,
 we anticipate the set of available models to expand
 as new methods and new modeling frameworks are developed.
 For instance, the current models all describe a small set of discrete, randomly mating populations,
 which are likely good approximations for deep-time population history,
 but may be less useful for methods describing dynamics of contemporary populations.
 \Stdpopsim's framework is sufficiently general that more realistic population models
-will be easily incorporated, once they are published.
-%DRS: trying to make this part a little less subjective. We were kind of painting the picture that
-%we have all the best/useful models and everything currently published but not included is not useful,
-%and I don't think this is justified.
-Our current set may also omit other widely used models that can be readily incorporated into \stdpopsim.
-Additional complications such as changing
-genome builds as assemblies improve must also be incorporated.
-As the collection of models and species grows, we hope to
-incorporate crowd-sourced community input
-to highlight the most appropriate models and genome annotations.
+will be easily incorporated, as they are published.
+
+%ADK: I cut this off here-- I think thais is a fine ending
```

Comment:
Made a sort of larger cut at the end of the discussion here... hoping to get some 👀 on this

Reply:
This bit was explicitly in here as a response to a reviewer request; I may put some of it back in.

Reply:
Adding
"
Additional aspects of the framework,
such as genome builds, will also continue to change
as improvements are made to our understanding of genome structure.
"